code/__DEFINES/is_helpers.dm

```diff
@@ -281,3 +281,4 @@ GLOBAL_LIST_INIT(book_types, typecacheof(list(
 #define is_unassigned_job(job_type) (istype(job_type, /datum/job/unassigned))
 
 #define isprojectilespell(thing) (istype(thing, /datum/action/cooldown/spell/pointed/projectile))
+#define is_multi_tile_object(atom) (atom.bound_width > 32 || atom.bound_height > 32)
```

Comment:
world.icon_size?

Reply:
yeah honestly. it might make movement mildly slower but that's ok